src/snippets/article.v1.yaml

```diff
@@ -89,6 +89,10 @@ properties:
                 minItems: 1
         required:
           - content
+    publicationHistory:
```

Comment:
Looking at `<event>` (https://jats.nlm.nih.gov/archiving/tag-library/1.2d2/element/event.html) and its intent, it allows you to add some additional information about a publication event (or non-publication event). It seems to me more closely aligned with the `accepted` and `received` dates found currently in the `<history>` tag. These dates are associated with all versions of the article and, although the preprint date would not change between versions, it may not be precise to associate a preprint with article itself, and might be better aligned with the `article-history.v1.yaml` somehow? I don't have a concrete example because it is not easy.